listenbrainz/webserver/templates/index/index.html

```diff
@@ -46,41 +50,42 @@ <h2>About us</h2>
         For more information about this project and its goals, please take a look at our
         <a href="{{ url_for('index.goals') }}">goals page</a>.
       </p>
-    </div>
-
-    <div class="col-md-5 col-lg-4">
-
-      <div id="stats" class="panel panel-default">
+      <div id="stats" >
         <div class="panel-heading"><strong>Data Statistics</strong></div>
         <div class="panel-body">
-
           <table class="table table-collapsed">
             <thead>
-              <tr>
-                <th>Description</th>
-                <th>Number</th>
-              </tr>
+            <tr>
+              <th>Description</th>
+              <th>Number</th>
+            </tr>
             </thead>
             <tbody>
             {% if listen_count %}
-              <tr>
-                <td>Number of Listens</td>
-                <td>{{ "{:,}".format(listen_count) }}</td>
-              </tr>
+            <tr>
+              <td>Number of listens</td>
+              <td>{{ listen_count }}</td>
+            </tr>
+            {% endif %}
+            {% if user_count %}
+            <tr>
+              <td>Number of users</td>
+              <td>{{ user_count }}</td>
+            </tr>
             {% endif %}
             </tbody>
           </table>
           <p style="text-align: center">
-            <small>updated every 5 minutes</small><br />
+            <small>Updated every 5 minutes!</small><br />
```

Comment:
Not sure this is needed (it's not that exciting that we update our stats :p )
```suggestion
            <small>Updated every 5 minutes</small><br />
```

Reply:
Lol. Right!